dreamy-backend/server.js

```diff
@@ -0,0 +1,59 @@
+const path = require('path');
+const express = require('express');
+const app = express();
+const mongoose = require('mongoose');
+
+// var users = require('./');
```

Comment:
You have some commented out code here.  What is it for?  Do we really need it in our finished product?